### DISCUSSION TITLE
Векторные схемы FSD светлая/тёмная

### DISCUSSION THREAD

**Discussion by iqhater** [feature-sliced/documentation#775] [DocDraft]
Приветствую команду FSD!

Перерисовал в svg формат схему FSD. Светлая и тёмная. 
Мой небольшой контирбут в сообщество.

Оставляю ссылки на публичный исходник в figma для свободного редактирования и два варианта схем в svg (светлую и тёмную)

Light: 
![fsd-schema-light](https://github.com/user-attachments/assets/9685c37d-fe10-4467-9190-8b39836a329a)

Dark: 
![fsd-schema-dark](https://github.com/user-attachments/assets/21c38049-99d9-4e48-bf64-1cdcf2dd63c7)

Figma source: https://www.figma.com/community/file/1445804842659690135/fsd-structure-schema

**Comment by illright** [comment]
Спасибо, но у нас они и так уже в фигме в векторе :) в доку залили растровую версию, потому что так проще было, и отличия минимальные

**Reply by iqhater** [reply]
О супер! Да, я с Вашей перерисовывал, думал нет в векторе.
